pkgs/development/python-modules/btrees/default.nix

```diff
@@ -5,27 +5,44 @@
 , zope_interface
 , transaction
 , zope_testrunner
+, python
 }:
 
 buildPythonPackage rec {
   pname = "BTrees";
   version = "4.9.2";
 
-  buildInputs = [ transaction ];
-  propagatedBuildInputs = [ persistent zope_interface ];
-  checkInputs = [ zope_testrunner ];
-
-  # disable a failing test that looks broken
-  postPatch = ''
-    substituteInPlace BTrees/tests/common.py \
-      --replace "testShortRepr" "no_testShortRepr"
-  '';
-
   src = fetchPypi {
     inherit pname version;
     sha256 = "d33323655924192c4ac998d9ee3002e787915d19c1e17a6baf47c9a63d9556e3";
   };
 
+  buildInputs = [
```

Comment:
```suggestion
  nativeBuildInputs = [
```
?

Reply:
Good question.

Reply:
It's actually a test dependency.

Reply:
I think there is `checkInputs`?

Reply:
Yep, updated in https://github.com/NixOS/nixpkgs/pull/128634/commits/dfd233ce45c08a9543742d69c6581d7683a6a376.